dspace-server-webapp/src/test/java/org/dspace/app/rest/BundleUploadBitstreamControllerIT.java

```diff
@@ -136,6 +136,71 @@ public void uploadBitstreamAllPossibleFieldsProperties() throws Exception {
 
     }
 
+    @Test
+    public void uploadBitstreamAllPossibleFieldsPropertiesButNoFile() throws Exception {
```

Comment:
here we have potentially the same issue described here https://jira.lyrasis.org/browse/DS-4278
for the item but related to the bitstream. I would recommend to create a separate ticket for that or extend the scope of DS-4278 to deal also with that if this PR  is merged soon